### ISSUE TITLE
Project does not load with 1.19.3

### ISSUE THREAD

**Issue by gonozalviii** [antlr/intellij-plugin-v4#595]
I just updated the plugin to version 1.19.3 and after the restart the project gets stuck in the loading bar. Downgrading or disabeling the plugins solves the Problem.

**Comment by imkiva** [comment]
I also suffer from this problem after I upgraded the antlr plugin and installed my own plugin. And I thought there was something wrong with my plugin 🤣

**Comment by parrt** [comment]
hi! @gonozalviii @imkiva can you folks tell me what operating system, jetbrains tool (and version), and java version you have installed?

**Comment by topia** [comment]
I also hit this problem. I think almost any 2022.3 (EAP) IDEs have affected this problem.
My environment:
```
IntelliJ IDEA 2022.3 RC (Ultimate Edition)
Build #IU-223.7571.58, built on November 17, 2022
Runtime version: 17.0.5+1-b653.14 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 11 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 6144M
Cores: 24

...

Kotlin: 223-1.7.20-release-201-IJ7571.58
```

**Comment by imkiva** [comment]
Thanks for the quick reply! This is my local configuration.

```
IntelliJ IDEA 2022.3 RC (Ultimate Edition)
Build #IU-223.7571.58, built on November 17, 2022

Runtime version: 17.0.5+1-b653.14 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 32768M
Cores: 32
Registry:
    run.processes.with.pty=false
    scala.erase.compiler.process.jdk.once=false

... <a lot of plugins including antlr4>

Kotlin: 223-1.7.20-release-201-IJ7571.58
```

**Comment by gonozalviii** [comment]
I also run the EAP Version

`
IntelliJ IDEA 2022.3 RC (Ultimate Edition)
Build #IU-223.7571.58, built on November 17, 2022

Runtime version: 17.0.5+1-b653.14 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 11 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 4000M
Cores: 20
`

**Comment by parrt** [comment]
Thanks...i am rebuilding using intel jdk 11 (though I'm on apple silicon).  Also making sure I have latest gradle. Will try on EAP though that might be an issue...

**Comment by parrt** [comment]
Weird.  Can't get 2022.3 to run (updated to jdk 17): `Caused by: org.gradle.internal.resolve.ModuleVersionNotFoundException: Could not find com.jetbrains.intellij.idea:ideaIC:2022.3.`

**Comment by mkw** [comment]
Unfortunately, the 1.9.3 version is still installed by default, no the 1.9.3-1 version, so if you update or install this into the EAP, projects fail to load. I wonder if there is any way to patch the 1.9.3 version in place?

**Comment by parrt** [comment]
I wonder if it's the intellij api change. we went from 1.6 to 1.10!  maybe i back that down and see?

**Comment by greenhull** [comment]
I'm seeing the same thing. in Intellij 2022.3 release candidate, software version: 

version IntelliJ IDEA 2022.3 RC (Ultimate Edition)
Build #IU-223.7571.58, built on November 16, 2022
Licensed to xxxx xxxx
Subscription is active until March x, 2023.
Runtime version: 17.0.5+1-b653.14 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 11 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 36
Registry:
    editor.focus.mode.color.light=e0861f
    ide.editor.tab.selection.animation=true
    ide.new.editor.tabs.vertical.borders=true
    ide.balloon.shadow.size=0
    editor.focus.mode.color.dark=e0861f

Non-Bundled Plugins:
    com.chylex.intellij.inspectionlens (1.1.0)
    org.sylfra.idea.plugins.linessorter (1.0.1)
    String Manipulation (9.6.1)
    MavenRunHelper (4.23.222.2964.0)
    GrepConsole (12.15.211.6693.1)
    izhangzhihao.rainbow.brackets (2022.3.4)
    com.chrisrm.idea.MaterialThemeUI (8.2.0)
    org.sonarlint.idea (7.2.0.56810)
    com.mallowigi (80.0.0)

Kotlin: 223-1.7.20-release-201-IJ7571.58

**Comment by parrt** [comment]
damn. Let me try backing down the api version.

**Comment by parrt** [comment]
Unfortunately backing down the API version doesn't work. It still feels to load with 2022.3. It should not allow you to actually load it though... I just got the warning that that plugin version it's not compatible. Did you load 1.19.3-1 or 1.19.3? @greenhull

**Comment by greenhull** [comment]
the one I'm seeing is 1.19.3, I'll uninstall and reinstall it and let you know.

**Comment by parrt** [comment]
I think that one will tell you that it doesn't work and won't let you load it. sorry. Hopefully when it goes "live" 2022.3 will work. I can check it then. The reality is I already work seven days a week maintaining Open source software in addition to my job so this one may go down and priority as I work on lab.antlr.org

**Comment by greenhull** [comment]
No way to back off the last release of Antlr V4 plugin?

**Comment by parrt** [comment]
oh. Maybe the previous version 1.18 will work yes. try it!

**Comment by greenhull** [comment]
how do I load it?

**Comment by parrt** [comment]
Maybe download it from intellij plugins web and install from the disk?

**Comment by greenhull** [comment]
I'll give it try, thanks

**Comment by greenhull** [comment]
Here's what I found, if you download Antl V4 1.19.2 from: https://plugins.jetbrains.com/plugin/7358-antlr-v4/versions and do an install from disk, it works (after doing an antlr V4 uninstall)

**Comment by parrt** [comment]
ok, good to know. That will give me some ammunition to look for differences... There should not be many changes so I'm surprised 1.19.3 doesn't work. If you have time to look at the differences that would be a big help...

**Comment by imkiva** [comment]
It seems the only change related to initialization is here: https://github.com/antlr/intellij-plugin-v4/compare/1.19.2...1.19.3#diff-aa2e7919b16b9fc2dcf48d797342388c37bc7f7ff518718ca1f3404129f2b846R118-R127

**Comment by imkiva** [comment]
Now I see the version of antlr4 is 1.19.3-1 in the Marketplace
![image](https://user-images.githubusercontent.com/9902643/203695107-ecf3106b-aede-4df8-b872-15ba3df227fc.png)

**Comment by gonozalviii** [comment]
I can confirm that this is the code, that breaks the loading. I'm not sure why it is there, since the filetype is associated in the plugin.xml.

**Comment by gonozalviii** [comment]
Also, moving this code out of `initComponents()` into `projectOpened()` does the reassign but does not break the loading.

**Comment by parrt** [comment]
amazing, @gonozalviii !!! thanks. i'll try to push another release

**Comment by parrt** [comment]
Hi. i uploaded 1.19.3-3 https://plugins.jetbrains.com/plugin/7358-antlr-v4  could you try it out? seems to work! @gonozalviii @imkiva thanks!

**Comment by parrt** [comment]
Hmm...many tests fail now:

```
CompositeException (2 nested):
------------------------------
[1]: Editor EditorImpl[null] hasn't been released:
[2]: Editor EditorImpl[null] hasn't been released:
------------------------------

CompositeException (2 nested):
------------------------------
[1]: com.intellij.openapi.util.TraceableDisposable$DisposalException: Editor EditorImpl[null] hasn't been released:
[2]: com.intellij.openapi.util.TraceableDisposable$DisposalException: Editor EditorImpl[null] hasn't been released:
------------------------------
```

Can't imagine that backout was the cause.

**Comment by greenhull** [comment]
It seems to work, Thanks

**Comment by parrt** [comment]
cool. can't figure out why tests are failing though. ugh.

**Comment by itzg** [comment]
This is the ticket where I'm waiting for the real fix

https://youtrack.jetbrains.com/issue/IDEA-306463

**Comment by greenhull** [comment]
possible same problem or new: exception in Antlr V4 1.19.3-3. stacktrace:
java.lang.RuntimeException: Cannot invoke (class=MyFileEditorManagerAdapter, method=selectionChanged, topic=FileEditorManagerListener)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeListener(MessageBusImpl.kt:639)
	at com.intellij.util.messages.impl.MessageBusImplKt.deliverMessage(MessageBusImpl.kt:395)
	at com.intellij.util.messages.impl.MessageBusImplKt.pumpWaiting(MessageBusImpl.kt:374)
	at com.intellij.util.messages.impl.MessageBusImplKt.access$pumpWaiting(MessageBusImpl.kt:1)
	at com.intellij.util.messages.impl.MessagePublisher.invoke(MessageBusImpl.kt:433)
	at jdk.proxy2/jdk.proxy2.$Proxy127.selectionChanged(Unknown Source)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl.lambda$fireSelectionChanged$26(FileEditorManagerImpl.java:1805)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl$6.lambda$run$0(FileEditorManagerImpl.java:1252)
	at com.intellij.openapi.util.ExpirableRunnable$1.run(ExpirableRunnable.java:17)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.lambda$doWhenFocusSettlesDown$3(FocusManagerImpl.java:173)
	at com.intellij.util.ui.EdtInvocationManager.invokeLaterIfNeeded(EdtInvocationManager.java:33)
	at com.intellij.ide.IdeEventQueue.ifFocusEventsInTheQueue(IdeEventQueue.java:176)
	at com.intellij.ide.IdeEventQueue.executeWhenAllFocusEventsLeftTheQueue(IdeEventQueue.java:129)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.doWhenFocusSettlesDown(FocusManagerImpl.java:169)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl$6.run(FileEditorManagerImpl.java:1251)
	at com.intellij.openapi.util.BusyObject$Impl$Simple.execute(BusyObject.java:105)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl.notifyPublisher(FileEditorManagerImpl.java:1248)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl.fireSelectionChanged(FileEditorManagerImpl.java:1805)
	at com.intellij.openapi.fileEditor.impl.EditorsSplitters.setCurrentWindow$lambda-19(EditorsSplitters.kt:647)
	at com.intellij.openapi.fileEditor.impl.EditorsSplitters.setCurrentWindow(EditorsSplitters.kt:656)
	at com.intellij.openapi.fileEditor.impl.EditorsSplitters$MyFocusWatcher.focusedComponentChanged(EditorsSplitters.kt:771)
	at com.intellij.openapi.wm.FocusWatcher._setFocused(FocusWatcher.java:159)
	at com.intellij.openapi.wm.FocusWatcher.setFocusedComponentImpl(FocusWatcher.java:154)
	at com.intellij.openapi.wm.FocusWatcher.focusGained(FocusWatcher.java:94)
	at java.desktop/java.awt.AWTEventMulticaster.focusGained(AWTEventMulticaster.java:228)
	at java.desktop/java.awt.AWTEventMulticaster.focusGained(AWTEventMulticaster.java:227)
	at java.desktop/java.awt.Component.processFocusEvent(Component.java:6537)
	at java.desktop/java.awt.Component.processEvent(Component.java:6404)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5022)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4854)
	at java.desktop/java.awt.KeyboardFocusManager.redispatchEvent(KeyboardFocusManager.java:1953)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.typeAheadAssertions(DefaultKeyboardFocusManager.java:1064)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.dispatchEvent(DefaultKeyboardFocusManager.java:688)
	at com.intellij.ide.IdeKeyboardFocusManager.lambda$dispatchEvent$0(IdeKeyboardFocusManager.java:32)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:113)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:615)
	at com.intellij.ide.IdeKeyboardFocusManager.dispatchEvent(IdeKeyboardFocusManager.java:32)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:4903)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4854)
	at java.desktop/sun.awt.SunToolkit$1.run(SunToolkit.java:516)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:779)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:730)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:724)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:749)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:909)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:756)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$5(IdeEventQueue.java:437)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:774)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:436)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:105)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:615)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:434)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:813)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:480)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
Caused by: java.lang.NullPointerException: Cannot invoke "String.equals(Object)" because the return value of "com.intellij.openapi.vfs.VirtualFile.getExtension()" is null
	at org.antlr.intellij.plugin.ANTLRv4PluginController.currentEditorFileChangedEvent(ANTLRv4PluginController.java:318)
	at org.antlr.intellij.plugin.ANTLRv4PluginController$MyFileEditorManagerAdapter.selectionChanged(ANTLRv4PluginController.java:659)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeMethod(MessageBusImpl.kt:655)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeListener(MessageBusImpl.kt:621)
	... 65 more

Intellij Info:

IntelliJ IDEA 2022.3 RC (Ultimate Edition)
Build #IU-223.7571.123, built on November 23, 2022
Licensed to Chester John
Subscription is active until March 9, 2023.
Runtime version: 17.0.5+1-b653.14 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 11 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 750M
Cores: 16
Registry:
    editor.focus.mode.color.light=e0861f
    ide.editor.tab.selection.animation=true
    ide.new.editor.tabs.vertical.borders=true
    ide.balloon.shadow.size=0
    editor.focus.mode.color.dark=e0861f

Non-Bundled Plugins:
    com.chylex.intellij.inspectionlens (1.1.0)
    org.antlr.intellij.plugin (1.19.3-3)
    org.sylfra.idea.plugins.linessorter (1.0.1)
    String Manipulation (9.6.1)
    MavenRunHelper (4.23.222.2964.0)
    GrepConsole (12.15.211.6693.1)
    izhangzhihao.rainbow.brackets (2022.3.4)
    com.chrisrm.idea.MaterialThemeUI (8.3.1)
    org.sonarlint.idea (7.2.0.56810)
    com.mallowigi (80.0.0)

Kotlin: 223-1.7.20-release-201-IJ7571.123

**Comment by parrt** [comment]
dang!  That looks like a new issue.

**Comment by geoff-m** [comment]
I am not having the same problems as OP as my project still loads, but I my ANTLR plugin (1.19.3-3) just threw a NPE with a similar stack trace. I am using Rider 2022.2.4 (build #RD-222.4459.9).
```
java.lang.NullPointerException: Cannot invoke "String.equals(Object)" because the return value of "com.intellij.openapi.vfs.VirtualFile.getExtension()" is null
	at org.antlr.intellij.plugin.ANTLRv4PluginController.currentEditorFileChangedEvent(ANTLRv4PluginController.java:318)
	at org.antlr.intellij.plugin.ANTLRv4PluginController$MyFileEditorManagerAdapter.selectionChanged(ANTLRv4PluginController.java:659)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeMethod(MessageBusImpl.kt:649)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeListener(MessageBusImpl.kt:625)
	at com.intellij.util.messages.impl.MessageBusImplKt.deliverMessage(MessageBusImpl.kt:399)
	at com.intellij.util.messages.impl.MessageBusImplKt.pumpWaiting(MessageBusImpl.kt:378)
	at com.intellij.util.messages.impl.MessageBusImplKt.access$pumpWaiting(MessageBusImpl.kt:1)
	at com.intellij.util.messages.impl.MessagePublisher.invoke(MessageBusImpl.kt:437)
	at jdk.proxy2/jdk.proxy2.$Proxy119.selectionChanged(Unknown Source)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl.lambda$fireSelectionChanged$31(FileEditorManagerImpl.java:1615)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl$7.lambda$run$0(FileEditorManagerImpl.java:1156)
	at com.intellij.openapi.util.ExpirableRunnable$1.run(ExpirableRunnable.java:17)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.lambda$doWhenFocusSettlesDown$3(FocusManagerImpl.java:174)
	at com.intellij.util.ui.EdtInvocationManager.invokeLaterIfNeeded(EdtInvocationManager.java:113)
	at com.intellij.ide.IdeEventQueue.ifFocusEventsInTheQueue(IdeEventQueue.java:180)
	at com.intellij.ide.IdeEventQueue.executeWhenAllFocusEventsLeftTheQueue(IdeEventQueue.java:133)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.doWhenFocusSettlesDown(FocusManagerImpl.java:164)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.doWhenFocusSettlesDown(FocusManagerImpl.java:170)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl$7.run(FileEditorManagerImpl.java:1155)
	at com.intellij.openapi.util.BusyObject$Impl$Simple.execute(BusyObject.java:105)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl.notifyPublisher(FileEditorManagerImpl.java:1152)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl.fireSelectionChanged(FileEditorManagerImpl.java:1615)
	at com.intellij.openapi.fileEditor.impl.EditorsSplitters.lambda$setCurrentWindow$10(EditorsSplitters.java:728)
	at com.intellij.openapi.fileEditor.impl.EditorsSplitters.setCurrentWindow(EditorsSplitters.java:737)
	at com.intellij.openapi.fileEditor.impl.EditorsSplitters$MyFocusWatcher.focusedComponentChanged(EditorsSplitters.java:885)
	at com.intellij.openapi.wm.FocusWatcher._setFocused(FocusWatcher.java:160)
	at com.intellij.openapi.wm.FocusWatcher.setFocusedComponentImpl(FocusWatcher.java:155)
	at com.intellij.openapi.wm.FocusWatcher.focusGained(FocusWatcher.java:95)
	at java.desktop/java.awt.AWTEventMulticaster.focusGained(AWTEventMulticaster.java:228)
	at java.desktop/java.awt.AWTEventMulticaster.focusGained(AWTEventMulticaster.java:227)
	at java.desktop/java.awt.Component.processFocusEvent(Component.java:6537)
	at java.desktop/java.awt.Component.processEvent(Component.java:6404)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5022)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4854)
	at java.desktop/java.awt.KeyboardFocusManager.redispatchEvent(KeyboardFocusManager.java:1964)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.typeAheadAssertions(DefaultKeyboardFocusManager.java:1064)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.dispatchEvent(DefaultKeyboardFocusManager.java:688)
	at com.intellij.ide.IdeKeyboardFocusManager.lambda$dispatchEvent$0(IdeKeyboardFocusManager.java:48)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:113)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:624)
	at com.intellij.ide.IdeKeyboardFocusManager.dispatchEvent(IdeKeyboardFocusManager.java:48)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:4903)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4854)
	at java.desktop/sun.awt.SunToolkit$1.run(SunToolkit.java:516)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:779)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:730)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:724)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:749)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:918)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:766)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:450)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:791)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:449)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:105)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:624)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:447)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:881)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:493)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
```

**Comment by parrt** [comment]
Hi. Could you open a separate bug for this?  Anything weird with your setup?  Empty files?  Weird file names?  Can you give a sample grammar and sequence to reproduce? Thanks!

**Comment by bjansen** [comment]
@geoff-m your issue is the same as #611, which was fixed today and will be released in version 1.20 of the plugin.

**Comment by bjansen** [comment]
So, from what I understand the original problem was caused by the fix for #569, which has been reverted and a new version 1.19.3-3 was released to address this problem. I guess this ticket can be closed then.
